OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -129,6 +131,16 @@ void ConnectionStateChanged(OrderManager om, string password, NetworkConnection
 			Game.LobbyInfoChanged += UpdateSpawnOccupants;
 			Game.BeforeGameStart += OnGameStart;
 			Game.ConnectionStateChanged += ConnectionStateChanged;
+			modData.MapCache.MapUpdated = (string oldUid, string newUid) =>
```

Comment:
When `MapUpdated` becomes an `event`, we're going to want to unsubscribe from the event handler when we're done.

Store a reference to the method in a field, and subscribe with `modData.MapCache.MapUpdated += trackMapUids;`

In the overridden `Dispose` method, also call `modData.MapCache.MapUpdated -= trackMapUids;`

Reply:
done